config/types.go

```diff
@@ -221,6 +221,39 @@ type Database struct {
 	Host    string
 	Port    int
 	Name    string
+
+	PostgresConnPool PostgresConnPool
```

Comment:
The current design depends on the specific database (postgres) and library (pgx)
Please use a single general `MaxConnections` for this issue